cmd/internal/cli/action_flags.go

```diff
@@ -805,6 +807,17 @@ var actionUnderlayFlag = cmdline.Flag{
 	Hidden:       false,
 }
 
+// --sharens
+var actionShareNSFlag = cmdline.Flag{
+	ID:           "shareNSFlag",
+	Value:        &shareNS,
+	DefaultValue: false,
+	Name:         "sharens",
+	Usage:        "mode for launching containers in the same namespace",
```

Comment:
I would like to see the usage also mention the need for the same parent process.  I suggest
```
share the namespace and image with other containers launched from the same parent process
```

Reply:
thanks! updated in the latest commit.